src/Native/Bootstrap/CppCodeGen.h

```diff
@@ -41,4 +41,33 @@ inline double __uint64_to_double(uint64_t v)
     return val.d;
 }
 
-#endif // __CPP_CODE_GEN_H
+#ifdef USE_PORTABLE_HELPERS
```

Comment:
I meant this ifdef to be in rhbinder.h. This file is only used when `USE_PORTABLE_HELPERS` is defined, so the ifdef does not need to be here.